src/test/java/ModelTest/ItemListTest.java

```diff
@@ -1,20 +1,21 @@
 package ModelTest;
 
-import Model.*;
-
 import static org.junit.Assert.*;
 
+import springboot.Model.Item;
 import org.junit.After;
 import org.junit.Before;
+import springboot.Model.Instructor;
+import springboot.Model.ItemList;
 
 public class ItemListTest {
     protected Item item1, item2;
     protected Instructor instructor1;
 
     @Before
     public void setUp(){
-        item1 = new Item("Item 1", "Description 1");
-        item2 = new Item("Item 2", "Description 2");
+        item1 = new Item("Model.Item 1", "Description 1");
```

Comment:
does the "Model." need to be here?

Reply:
No, this was probably from when I moved the packages around

Reply:
okay so I would remove it then, it's also in 2 other spots